src/components/Geolocation/Geolocation.scss

```diff
@@ -0,0 +1,80 @@
+.tm-geolocation {
+  cursor: pointer;
+  height: $btn-size-base;
+  width: $btn-size-base;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  text-align: center;
+  font-size: 20px;
+  color: white;
+  background-color: $btn-primary-color;
+  transition: background-color 0.5s ease;
+
+  &:hover {
+    background-color: $btn-primary-color-hover;
+  }
+
+  &.blink {
+    -webkit-animation-name: blinking;
+    -webkit-animation-duration: 3s;
+    -webkit-animation-timing-function: linear;
+    -webkit-animation-iteration-count: infinite;
+    -moz-animation-name: blinking;
+    -moz-animation-duration: 3s;
+    -moz-animation-timing-function: linear;
+    -moz-animation-iteration-count: infinite;
```

Comment:
No need of prefixed version of properties. All browser supports it now and if we need it we should use Autoprefixer or something else to generate it automatically.